libs/core/hardware/include/hpx/hardware/timestamp/bgq.hpp

```diff
@@ -17,11 +17,21 @@
 
 #include <cstdint>
 
+#include <hpx/config.hpp>
```

Comment:
Could you put this include above the standard library includes. This is consistent with how we do it elsewhere.